user_view.html

```diff
@@ -71,7 +73,7 @@ <h2> Your Recurring Donations </h2>
 <h2>Donation History </h2>
 <ul>
 {% for donation in donations %}
-<li>{{ donation.amt }} on {{ donation.created_at|date:"m/d/Y" }}  {% if donation.recurring %} (Recurring) {% endif %}
+<li>{{ donation.amt }} on {{ donation.created_at|date:"m/d/Y" }} {% if 'ActBlue' in donation.account %}via <a href="https://secure.actblue.com/signin">ActBlue</a>{% endif %} {% if donation.recurring %} (Recurring) {% endif %}
```

Comment:
Should we make this open in new tab? Maybe question for @rachelhbd

Reply:
Hmm, maybe that's a good idea since it's an external site. If you'd advise that let's do it!